superset-frontend/src/explore/components/controls/ConditionalFormattingControl/FormattingPopoverContent.tsx

```diff
@@ -38,10 +40,18 @@ const JustifyEnd = styled.div`
   justify-content: flex-end;
 `;
 
+export const hexToRGBString = (hex: string) => {
```

Comment:
We already have a `hexToRgb` in `src/utils/colorUtils` 😉

Reply:
Why yes there is... :facepalm: 

I grepped the code and only found one buried in a plugin somewhere.